includes/class-wcpdf-admin.php

```diff
@@ -28,6 +28,10 @@ public function __construct()	{
 			add_filter( 'manage_woocommerce_page_wc-orders_columns', array( $this, 'add_invoice_columns' ), 999 ); // WC 7.1+
 			add_action( 'manage_woocommerce_page_wc-orders_custom_column', array( $this, 'invoice_columns_data' ), 10, 2 ); // WC 7.1+
 			add_filter( 'manage_woocommerce_page_wc-orders_sortable_columns', array( $this, 'invoice_columns_sortable' ) ); // WC 7.1+
+            // Fix sortable issue when HPOS is on.
```

Comment:
HPOS is now the default for new stores, so maybe we shouldn't call this a fix?

Reply:
Good point. I have removed it here: https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/pull/700/commits/3b58258afb8207515131f32d04b2a85f4796f50f.